src/main/java/com/example/KitchenProject/RecipeController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.KitchenProject;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class RecipeController {
+    List <Recipe> recipes = Arrays.asList(
```

Comment:
```suggestion
    List<Recipe> recipes = Arrays.asList(
```
Please remember about formatting your code before committing. Intellij auto format is your friend:)